mmtrack/datasets/mot_challenge_dataset.py

```diff
@@ -29,11 +36,13 @@ class MOTChallengeDataset(CocoVideoDataset):
 
     def __init__(self,
                  visibility_thr=-1,
+                 interpolate_tracklet=None,
```

Comment:
tracklet to track?

Reply:
I name it as `interpolate_track_cfg`, since we have a function named as `interpolate_tracks`